### DISCUSSION TITLE
'open with' does't do anything

### DISCUSSION THREAD

**Discussion by johannesfoltt** [jarun/nnn#1689] [Q&A]
Hi!
I am new to nnn and I ran into a problem for which I cannot find a solution.
I want to use o (open with) to open a pdf file, so that I can decide whether to open it with Xournal++ or with a pdf-reader.
When I do that, I'm taken to the shell where I have to enter the program I want to use.
But no matter what I put in, I always land back in nnn and nothing else happens.
What am I doing wrong?

**Comment by jarun** [comment]
Seems to be a local issue you need to debug yourself.
I use `zathura` and it works fine. Make sure you press `g` in the next screen if it's a GUI pdf opener.

**Comment by sabitm** [comment]
I'm also experiencing the same issue. This happens when I'm using the `nnn` package from the official Arch repository (Also tried the `nnn-git` and `nnn-nerd` from AUR but the issue still persist. Hmm, weird). Downloading the pre-compiled binary in Release works for me.

**Comment by Logicdemon** [comment]
I am having the same issue too for some time. Running the latest version from the Arch repository. Pressing o and then typing say gimp or ffplay results in nothing happening.

Thanks

**Comment by jarun** [comment]
Can't repro locally with compiled binary from latest source.
Please contact the Arch maintainer.

**Reply by Logicdemon** [reply]
Thanks, I will report it to the maintainer.

Cheers
